.github/workflows/wheels-build.yaml

```
@@ -188,32 +188,32 @@ jobs:
         shell: bash -leo pipefail {0}
 
       - name: Get package name
-        if: inputs.wheel_type != ''
+        if: inputs.package-type != ''
         env:
-          WHEEL_TYPE: ${{ inputs.wheel_type }}
-          WHEEL_NAME: ${{ inputs.wheel_name }}
-          PURE_WHEEL: ${{ inputs.pure_wheel }}
+          PACKAGE-TYPE: ${{ inputs.package-type }}
+          WHEEL-NAME: ${{ inputs.wheel-name }}
+          PURE-WHEEL: ${{ inputs.pure-wheel }}
```

```suggestion
          PACKAGE_TYPE: ${{ inputs.package-type }}
          WHEEL_NAME: ${{ inputs.wheel-name }}
          PURE_WHEEL: ${{ inputs.pure-wheel }}
```

I'm not sure that `-` characters are valid for environment variable names.

When I ran `TEST-VAR=1` in `bash`, I got an error.

Can we switch the environment variables to use underscores? I realize this will cause them to be inconsistent with the `input` key names.

---

I agree, the environment variable form should have underscores.

---

Oh I didn't know that. Changed the environment variables to use underscores now.